### PR TITLE
Feat: Code of conduct

### PR DESCRIPTION
**Pull request by Amanya-Brian**

### CODE REVIEW

**Review by basiima — APPROVED** [review]
I approve the changes.

File: pages/code-of-conduct.js [hunk 1]
```diff
@@ -1,3 +1,118 @@
+import RootLayout from "@/app/layout";
+
+export const metadata = {
```
Comment (HassanBahati) [inline pages/code-of-conduct.js:3]: @Amanya-Brian this is going to break the app build

Comment (HassanBahati) [reply pages/code-of-conduct.js:3]: image

**Review by HassanBahati — APPROVED** [review]
@Amanya-Brian i have tried to fix the errors and also made some changes to files location.

the ui is the same as you had made it

i would suggest we create pages inside the `app directory` this is for performance as the files in their are server components by default

please note that every route expects a file called `page.js` i would prefer to have it these pages and components with `.jsx` extension

@basiima

**Comment by basiima** [comment]
@HassanBahati do you mean all components should be .jsx or only those in the app directory?

**Comment by HassanBahati** [comment]
> @HassanBahati do you mean all components should be .jsx or only those in the app directory?

its not a must....all will still work fine...they are interchangeable...

what i base on for my consideration when deciding what to put into a `.js`or a `.jsx` file type is the content. Since JSX isn't standard JavaScript I could argue that anything that is not "plain" JavaScript should go into its own extensions of `.js` and where html is together with Javascript in `.jsx` for `JSX`

you could check out this from react docs https://legacy.reactjs.org/docs/introducing-jsx.html#why-jsx

**Pull request merged**